### ISSUE TITLE
L'editeur ne load pas correctement les rigidbody

### ISSUE THREAD

**Issue by cesarl** [Another-Game-Engine/AGE#546]
Il les place n'importe ou

**Comment by paulbaron** [comment]
Avec bullet...

**Comment by herpec-j** [comment]
Oui, c'est normal avec Bullet, il faudrait passer l'éditeur sous PhysX à la
limite.

On Tue, Aug 18, 2015, 2:33 AM Paul Baron <notifications@github.com> wrote:

> Avec bullet...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Another-Game-Engine/AGE/issues/546#issuecomment-132004840>
> .
>
-- 

Mr. HERPÊCHE Jonathan
Software Engineer at Agueris
+33 (0)6 20 05 13 72
